### DISCUSSION TITLE
Sugarcane fallow

### DISCUSSION THREAD

**Discussion by daSilva5** [APSIMInitiative/ApsimX#9413] [Q&A]
Hi,

There is some comments in the Sugar Management script about fallow:

//two below are used for rotations. Remove this. I don't think ApsimX does rotations like this.
        private int nextcrop;
        private int fallowin;

        
        [EventSubscribe("StartOfSimulation")]
        private void OnStartOfSimulation(object sender, EventArgs e)
        {
            //Rotation variables. Not sure if these are still needed in ApsimX rotations
            nextcrop = 0;
            fallowin = 0;
        }

I would like to know if it is actually doing anything at the moment, such as connecting with Surface Organic Matter module. Or are these two just hanging in there?

**Comment by BrianCollinss** [comment]
Do you see those two being assigned to any other internal state variables, or use in any other ways in the manager script?

**Reply by daSilva5** [reply]
> Do you see those two being assigned to any other internal state variables, or use in any other ways in the manager script?
No, this is the only time these variables appear.

**Reply by BrianCollinss** [reply]
In NextGen, a local variable is only important if at some point its value is assigned to an internal state variable, or it is a public property and is reported by Reports. So, you can ignore/delete them, IMO. You can see if outputs change when you delete/comment them out.

**Reply by daSilva5** [reply]
Thanks!
